src/Restaurant.js

```diff
@@ -1,17 +1,39 @@
+import { useState } from "react";
 import React from "react";
 
-const Restaurant = () => {
+const RestaurantButton = props => {
+  return (
+    <button onClick={props.orderOne} className="btn btn-primary">
+      Add
+    </button>
+  );
+};
+
+const Order = props => {
   const pizzas = 0;
+  const [order, setOrder] = useState(pizzas);
```

Comment:
I would just pass 0 to useState here - it doesn't need to be assigned to the pizzas var first. I would also break each componant out into it's own file for neatness. Works as described though